### PR TITLE
add custom error page for unhandled errors/500

### PR DESCRIPTION
**Pull request by auxibee**
Try http://localhost:3000/discussions/400044 to see page

### CODE REVIEW

File: client/app/root.tsx [hunk 1]
```diff
@@ -37,6 +37,38 @@ export const links: LinksFunction = () => [
...
+			<body>
+				<div className="flex items-center max-w-4xl  mx-auto h-screen justify-center">
+					<div className="flex flex-col items-center space-y-3">
+						<img src="/sym.svg" width={240} className="block" alt="Compa" />
+						<h1 className="font-bold text-3xl">Sorry, Unexpected Error</h1>
+						<p className="text-gray-500 max-w-2xl text-center">
+							We are facing an internal server error. Our experts are trying to
+							fix the problem. Please try again or wait for sometime
+						</p>
+						<a
```
Comment (blackmann) [inline client/app/root.tsx]: There's is an `<Anchor />` component you can use.

**Comment by blackmann** [comment]
Below are the designs for the error boundary:

404s aren't really a bad error so we shouldn't treat them as runtime or server errors. Could you make those distinctions.
For errors that have status codes, we prefix the heading with the status code.

Let's log those errors to PostHog too. See: https://posthog.com/docs/getting-started/send-events#sending-custom-properties-on-an-event. You can capture relevant details along as you see fit.

In addition, we should show the footer on this page too.

Instagram post - 7
Instagram post - 6

**Comment by auxibee** [comment]
Okay I Will do That

**Comment by blackmann** [comment]
Thanks for this @auxibee 

I change the icon to the mono one and add a dev mode log for stacktraces.

**Comment by auxibee** [comment]
Thank you too.

**Pull request merged**